### ISSUE TITLE
links to the dataset does not exist

### ISSUE THREAD

**Issue by hex41434** [myavartanoo/PolyNet_PyTorch#1]
Hi, Thanks for the repo you published on github, I tried to use the links [PTQ] and [√3-subdivision] and seems the links are broken. could you please fix this? 

Best

**Comment by myavartanoo** [comment]
Hi, Thanks for your interest. The links are updated now so please check them again.

**Comment by hex41434** [comment]
Great, Thanks a lot! now I have access to the folders on Google drive. would you mind if I ask you to share the entire folder as a zip file too?

**Comment by myavartanoo** [comment]
Google drive has an option that makes a zip file from the entire folder then you can download the zip file.

**Comment by hex41434** [comment]
yes of course! I download the "√3-subdivision" folder as a large zip file from gdrive, but with PTQ folder, I was not successful.
